docs/installation.md

```
@@ -0,0 +1,79 @@
+
+```bash
+ansible-galaxy collection list wiphoo.frp
+```
+
+## Quick Start Playbook
+
+Minimal playbook examples to install server and client components.
+
+```yaml
+- name: Install FRP Server
+	hosts: frp_servers
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)